scripts/spec_xml/paths.py

```diff
@@ -22,9 +22,8 @@
 
 class Branch(Enum):
     MASTER = "master"
-    V1_3 = "v1_3"
```

Comment:
in_progress directory is required for ongoing work. Please do not remove. It has been re-named, but the directory name is changeable on the command line.

Are you attempting to pull a new data model scrape for ongoing work?